CHANGELOG-3.0.md

```diff
@@ -0,0 +1,119 @@
+# Release notes for master
```

Comment:
Can we put "3.0.0" at the top? It's slightly more work to figure out which release this corresponds to from the "since X" line.

Reply:
Fixed, but it's manual edit and likely to be lost when regenerated next time.